packages/core/src/connection.ts

```diff
@@ -22,8 +22,8 @@
  * @private
  */
 class Connection {
-  id: string
-  databaseId: string
+  id: string = ""
+  databaseId: string = ""
```

Comment:
This fixes:
`"Property 'id' has no initializer and is not definitely assigned in the constructor."`